docs/core-manage-plugins.md

```diff
@@ -1,24 +1,25 @@
-Plugins are how asdf understands handling different packages.
+Plugins are how asdf knows how to handle different packages.
 
-See [All plugins](plugins-all) for the plugins repository which lists all asdf plugins we know.
-
-See [Creating Plugins](plugins-create) for the super-simple API for supporting more languages.
+See [Creating Plugins](plugins-create) for the plugin API used to support more languages.
 
 ## Add
 
+Add via the short-name association in the plugins repository:
+
 ```shell
 asdf plugin add <name>
 # asdf plugin add erlang
 ```
 
-If the plugin you want to install is not part of the plugins repository, you
-can add it using its repository URL:
+If not in the short-name repo, you can add it using its repository URL:
```

Comment:
nitpick: you can add it via URL regardless, and the plugins repository displays URLs for this purpose. It might be better to just say:

"Alternatively, you can add a plugin using its repository URL:"

Reply:
I swapped the order so the Git URL method is first and clarified the language based on your feedback, thanks.